addons/test_event_full/tests/test_event_mail.py

```diff
@@ -1,15 +1,51 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from datetime import timedelta
+from datetime import datetime, timedelta
 
 from odoo.addons.mail.tests.common import MockEmail
 from odoo.addons.sms.tests.common import MockSMS
+from odoo.addons.test_event_full.tests.common import TestEventFullCommon
 from odoo.addons.test_event_full.tests.common import TestWEventCommon
 from odoo.exceptions import ValidationError
 from odoo.tools import mute_logger
 
-class TestTemplateRefModel(TestWEventCommon):
+class TestTemplateRefModel(TestEventFullCommon, TestWEventCommon):
```

Comment:
Do you need this new class inheritance ? Isn't it simpler to create an event (type) manually in the test with a mail and a sms template linked ? Would be more explicit I think.